packages/loader/src/ShaderChunkLoader.ts

```
@@ -0,0 +1,44 @@
+  LoadItem,
+  Loader,
+  ResourceManager,
+  ShaderFactory,
+  resourceLoader,
+  // @ts-ignore
+  ShaderLib
+} from "@galacean/engine-core";
+import { PathUtils } from "./PathUtils";
+
+@resourceLoader("ShaderChunk", ["glsl"])
+class ShaderChunkLoader extends Loader<void[]> {
```

Biome

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)